components/popover/popover-mixin.js

```diff
@@ -23,6 +23,11 @@ export const PopoverMixin = superclass => class extends superclass {
 			 * @type {boolean}
 			 */
 			noAutoClose: { type: Boolean, reflect: true, attribute: 'no-auto-close' },
+			/**
```

Comment:
I'v added this attribute in the standard way, but in a future PR I will switch to some type of `super.configure({ ... })` method to avoid conflicts with dropdown and tooltip. I'll do that separately to avoid any confusion with this PR.